src/main/cpp/src/ftos_converter.cuh

```diff
@@ -1149,6 +1149,57 @@ __device__ inline int compute_f2s_size(float value)
   return f2s_size(v, sign);
 }
 
+//===== special inf handling for json =====
+
+__device__ inline int copy_special_str_json(char* const result,
+                                            bool const sign,
+                                            bool const exponent,
+                                            bool const mantissa)
+{
+  // no NaN in json
+  if (exponent) {
+    if (sign) {
+      memcpy(result, "\"-Infinity\"", 11);
```

Comment:
Nitpick: Feel free to ignore:

I had to count the characters to make sure `11` is right. :]
One trick is to use `sizeof("\"-Infinity\"") - 1`.  That is `constexpr` and should yield `11`.
I suspect `strlen("\"-Infinity\"")` is more readable, but I'm not sure it's `constexpr`.

Reply:
Will do in a follow-on.